app/assets/stylesheets/components/_article.scss

```diff
@@ -0,0 +1,31 @@
+.article-intro {
+  background-color: #f3f7fa;
```

Comment:
also changing this color to be what's in the designs instead of the closest USWDS one...